src/cmd-init

```diff
@@ -26,14 +27,18 @@ Usage: coreos-assembler init --help
   If specified, SUBDIR is a subdirectory of the git repository that should
   contain manifest.yaml and image.yaml (or image.ks).
 
+  Use --extrarepos for builds that need .repo files and content sets which are not in GITCONFIG.
```

Comment:
The commit message and this say `--extrarepos`, but the actual `getopt` option added is `--extrepos`.

Reply:
It should just be `--extrepos` for both. I'll fix that.